app/kubernetes-ingress-controller/1.0.x/guides/using-mtls-auth-plugin.md

```diff
@@ -57,6 +57,9 @@ Note that a self-signed CA certificate is being used for the purpose of this
 guide. You should use your own CA certificate that is backed by
 your PKI infrastructure.
 
+**The example here is only used to show the YAML format of `Secret` resource for CA certificate. DO NOT directly use the certificate here.
+You should use your own CA certificate, or generate a self-signed certificate for testing.**
```

Comment:
Can we provide an example of how to generate a self-signed certificate to help them get started?

Reply:
OK, I'd find an example of how to generate a self-signed CA certificate and client key/certificate pairs.